targets/ice40_hx8k_b_evn/base.py

```diff
@@ -49,10 +50,13 @@ class BaseSoC(SoCCore):
     )
     csr_map_update(SoCCore.csr_map, csr_peripherals)
 
-    mem_map = {
+    mem_map_overlay = {
```

Comment:
@mithro - do you want to keep this commit? Most of the changes taken from here are reverted in the following commit anyway.

Reply:
I mean **this** commit: https://github.com/timvideos/litex-buildenv/pull/348/commits/cf423281d98ddd3ac5bda4cd225c10f71053b9ab